### ISSUE TITLE
redraw issue on evolution

### ISSUE THREAD

**Issue by joakim-tjernlund** [mate-desktop/mate-themes#195]
evolution has the same issue as yelp had before" Gtk+-3.20 all themes: fix redraw issue with yelp" 
Will this fix help ?
Can we have a new release please?

**Comment by raveit65** [comment]
You can simply copy the change of the commit in gtk-widgets.css of your theme + reload the theme.
Than you can answer your question for yourself.
Sorry, i don't use evolution.
....new releases are on my TODO list.

**Comment by joakim-tjernlund** [comment]
So I added the change in TOK and the horiz. scrollbar returned :)

One thing though, just moving the mouse in a window with a horiz. scrollbar
killed the window and evo. printed:
```
Something has gone wrong when displaying the message.
A WebKitWebProcess crashed when displaying the message. You can try again by moving to another message and back. If the issue persists, please file a bug report in the GNOME bugzilla.
```
net-libs/webkit-gtk-2.16.6

**Comment by joakim-tjernlund** [comment]
The same thing happens with the just released 3.22.14 themes.
I guess webkit does not like the new horiz. scrollbar theming :(

**Comment by raveit65** [comment]
I know that webkitgtk4 is very buggy + having security issues in fedora the last half year.
I suggest to open a report at gnome bugzilla as theme code is valid.
